From 1d7cd501a5718d9b28d1afff505759871650dd58 Mon Sep 17 00:00:00 2001
From: ran shalit <rshalit@habana.ai>
Date: Tue, 16 Nov 2021 12:50:05 +0200
Subject: [PATCH] [SW-62501] hl: gaudi: add boot_info write

As part of adding write capability, gaudi2 onix
added with CONFIG_BOOTINFO_LOCAL_BASE, as there
is no defconfig specific for ONIX.

Change-Id: I79c285b21adf4d01df621d79258553d8a1312f3c
Signed-off-by: ran shalit <rshalit@habana.ai>
---

diff --git a/board/habanalabs/gaudi2/uboot_onix.fragment b/board/habanalabs/gaudi2/uboot_onix.fragment
index c52b7a3..c85e2a5 100644
--- a/board/habanalabs/gaudi2/uboot_onix.fragment
+++ b/board/habanalabs/gaudi2/uboot_onix.fragment
@@ -10,3 +10,4 @@
 CONFIG_FW_MAX_SIZE=0x20000
 CONFIG_SYS_MALLOC_EXTRA=0x10000
 CONFIG_SYS_TEXT_BASE=0x8200100000
+CONFIG_BOOTINFO_LOCAL_BASE=0x8200000000
